front/lib/workspace_usage.ts

```diff
@@ -440,7 +440,7 @@ export async function getAssistantUsageData(
   if (!mentions.length) {
     return "No data available for the selected period.";
```

Comment:
This string is parsed in agent_usages.ts ( `parseInt(agentUsage, 10)` ), which gets a NaN in this case